build-tools/packages/build-tools/src/common/utils.ts

```diff
@@ -180,11 +180,12 @@ export function isSameFileOrDir(f1: string, f2: string) {
 }
 
 /**
- * Execute a command. If there is an error, print error message and exit process
+ * Execute a command. If there is an error, throw.
```

Comment:
Do all callers expect a throw? Wondering about fluid-build in particular.

Reply:
Don't know about expectations, but that is what has [always] been happening.